pulp_container/app/tasks/untag.py

```diff
@@ -9,7 +9,7 @@ def untag_image(tag, repository_pk):
     repository = Repository.objects.get(pk=repository_pk).cast()
     latest_version = repository.latest_version()
 
-    tags_in_latest_repository = latest_version.content.filter(pulp_type="container.tag")
+    tags_in_latest_repository = latest_version.content.filter(pulp_type=Tag.get_pulp_type())
 
```

Comment:
git grep shows that there are some harcoded types in the viewsets.py can you update those also?

Reply:
yup